applications/add_unit_to_parameter_in_db.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to add a unit to parameters in the DB.
+
+    This application should not be used by anyone but expert users and only in extreme cases.
+    Therefore, no additional documentation about this applications will be given.
```

Comment:
I don't think this is the right thing to do - safety by not giving documentation? Doesn't this require write access to the DB, which most users don't have?

Reply:
As discussed, moved this application to a sub directory called `db_development_tools` for specific tools like that which do not require documentation.

Technically the API user can make these changes as well. We also have a read-only user but we do not use it (maybe we should start).